src/components/uploads/ImageUploader.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useRef } from "react";
+
+export const defaultImg =
+  "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR2Riw8iKZ4n4XAI3zEc0ShoGo1S5Z-oQdRWw&usqp=CAU";
+
+type ImageUploaderPropsType = {
+  buttonImg?: string;
+  onChange?: (_files: FileList | null) => void;
+  shape?: "rounded" | "circle" | "square";
+  size?: number;
+};
+
+const ImageUploader = ({
+  buttonImg = defaultImg,
+  onChange,
+  shape = "rounded",
+  size = 200,
+}: ImageUploaderPropsType) => {
+  const [image, setImage] = useState(buttonImg);
+  const fileInput = useRef<HTMLInputElement | null>(null);
+
+  const uploadHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const target = e.target as HTMLInputElement;
+    const file = target.files![0];
+    if (file) {
+      if (onChange) onChange(target.files);
+      setImage(file.toString());
+      previewImage(file);
+    } else {
+      setImage(defaultImg);
+      previewImage(null);
+    }
+  };
```

Comment:
공백만 있어도 훨씬 읽기 편할 것 같아요..!
```suggestion
    const target = e.target as HTMLInputElement;
    const file = target.files![0];
    
    if (file) {
      if (onChange) onChange(target.files);
      
      setImage(file.toString());
      previewImage(file);
    } else {
      setImage(defaultImg);
      previewImage(null);
    }
  };
```

Reply:
헉 좋은 지적 감사합니다! 수정하겠습니다.